rest-api-spec/src/main/resources/rest-api-spec/test/search/330_fetch_fields.yml

```diff
@@ -255,3 +255,50 @@ setup:
   - is_true: hits.hits.0._id
   - is_true: hits.hits.0._source
   - match: { hits.hits.0.fields.date.0: "1990/12/29" }
+
+---
+"Test skip malformed values":
+  - skip:
+      version: " - 7.99.99"
+      reason: "support for skipping malformed values is not yet backported to 7.x"
+  - do:
+      indices.create:
+        index:  test
+        body:
+          settings:
+            index.number_of_shards: 1
+          mappings:
+            dynamic: false
+
+  - do:
+      index:
+        index:  test
+        id:     1
```

Comment:
Maybe add `refresh: true` here and skip the explicit refresh below? Just saves a few lines.